main/src/ca/uwaterloo/flix/language/phase/unification/Substitution.scala

```diff
@@ -166,94 +163,4 @@ case class Substitution(m: Map[Symbol.TypeVarSym, Type]) {
 
     Substitution(newTypeMap.toMap) ++ this
```

Comment:
Maybe we should rename Substitution to `TypeEnv` (to mirror RigidityEnv) at some point.